### ISSUE TITLE
Videos containing topics related to suicide or selfharm are blocked and tapping to retry doesnt unblock them

### ISSUE THREAD

**Issue by Nyx755** [qnblackcat/uYouPlus#385]
if you try to play a video which has a warning before it like https://youtu.be/zOujzvtwZ6M u get the warning but that warning doesn't go away like it should. 

"The following content may contain topics related to suicide or self-harm.
Tap to retry"

this is not a problem with the unmodified youtube app and the warning looks different. the account is in both screenshots the same so this shouldn't be a account related problem. 

uyou newest version 
![8BE821B2-FAF7-4D5D-A111-5685618B9FE5](https://user-images.githubusercontent.com/51517828/185100364-6c0b2960-6dda-4449-8c4f-5565270159e4.jpeg)
unmod 17.32.2
![DA1DAE0D-6877-462A-914F-822243150550](https://user-images.githubusercontent.com/51517828/185100387-17eaaab3-8cd6-4c5c-9dd4-8053af10954f.jpeg)

**Comment by qnblackcat** [comment]
Duplicate of https://github.com/qnblackcat/uYouPlus/issues/250
